nautobot/ipam/api/nested_serializers.py

```diff
@@ -20,8 +21,18 @@
 
 class IPFieldSerializer(serializers.CharField):
     def to_representation(self, value):
+        """Convert internal (IPNetwork) representation to API (string) representation."""
         return str(value)
 
+    def to_internal_value(self, value):
+        """Convert API (string) representation to internal (IPNetwork) representation."""
+        try:
+            return netaddr.IPNetwork(value)
+        except netaddr.AddrFormatError:
+            raise serializers.ValidationError("Invalid IP address format: {}".format(value))
+        except (TypeError, ValueError) as e:
+            raise serializers.ValidationError(e)
```

Comment:
I'm surprised this was necessary since the underlying model field already knows how to convert the string into a the internal type. Curious what your experience was!

Reply:
It's basically there to ensure that we raise a `serializers.ValidationError` (which gets mapped to a HTTP 400 response) instead of an uncaught `netaddr.AddrFormatError` (which causes an HTTP 500 response)